beacon-chain/powchain/service.go

```diff
@@ -616,6 +616,9 @@ func (s *Service) processBlockHeader(header *gethTypes.Header) {
 // batchRequestHeaders requests the block range specified in the arguments. Instead of requesting
 // each block in one call, it batches all requests into a single rpc call.
 func (s *Service) batchRequestHeaders(startBlock, endBlock uint64) ([]*gethTypes.Header, error) {
+	if startBlock >= endBlock {
```

Comment:
```suggestion
	if startBlock > endBlock {
```
we should be allowing singular queries too, there might be cases where our batch request is only for 1 header.